src/components/challenge/ChallengeDropdown.jsx

```diff
@@ -16,6 +26,50 @@ const ChallengeDropdown = ({ onOptionChange }) => {
     setIsOpen(false);
   };
 
+  const applyHandler = () => {
+    const appliedFilters = filter.filter((_, index) => selectedFilters[index]);
+    const data = {
+      filter: appliedFilters,
+      docType: selectedDocType,
+      progress: selectedProgress,
+    };
+
+    if (docType[selectedDocType] === '공식문서') {
+      // console.log('OFFICIAL');
+      data.docType = 'OFFICIAL';
+    } else if (docType[selectedDocType] === '블로그') {
+      // console.log('BLOG');
+      data.docType = 'BLOG';
+    }
+    if (progress[selectedProgress] === '진행중') {
+      // console.log('true');
+      data.progress = true;
+    } else if (progress[selectedProgress] === '마감') {
+      // console.log('false');
+      data.progress = false;
+    }
+    // console.log(docType[selectedDocType]);
+    // console.log(appliedFilters);
+    // console.log(progress[selectedProgress]);
+    console.log(data);
+  };
```

Comment:
```suggestion
 const applyHandler = () => {
  const appliedFilters = filter.filter((_, index) => selectedFilters[index]);
  const data = {
    filter: appliedFilters,
    docType: docType[selectedDocType] === '공식문서' ? 'OFFICIAL' : 'BLOG',
    progress: progress[selectedProgress] === '진행중' ? true : false,
  };

  console.log(data);
};
```

`docType` 과 `progress` 관련 조건문을 삼항 연산자를 사용해보는건 어떨까요? 🙄